proto/recon.proto

```diff
@@ -143,6 +145,8 @@ message RecognizePlacesRequest {
 message RecognizePlacesResponse {
   message Place {
     string dcid = 1;
+    // All types of the entity. Ordered by dominant place type.
```

Comment:
nit: All place types?

Reply:
This will not just be place types, it may also be types like "Gene" or "Drug", etc. But if there is a "dominant place type", then that will still be at the front of the list